.gitignore

```diff
@@ -8,3 +8,4 @@
 /build
 /docs/_build
 coverage.xml
+venv/
```

Comment:
I'd prefer we not add things here. You should use a global gitignore if there's a way you're working that is different from the project conventions